cairo/src/context.rs

```diff
@@ -727,56 +732,40 @@ impl Context {
 
     #[doc(alias = "cairo_font_extents")]
     pub fn font_extents(&self) -> Result<FontExtents, Error> {
-        let mut extents = FontExtents {
-            ascent: 0.0,
-            descent: 0.0,
-            height: 0.0,
-            max_x_advance: 0.0,
-            max_y_advance: 0.0,
-        };
+        let mut extents = FontExtents::new();
```

Comment:
```suggestion
        let mut extents = MaybeUninit::uninit();
```

or `::zeroed()` maybe?

I don't think we need a constructor for this type at all, do we?

Reply:
We don't but I found it better this way.

Reply:
It's unnecessarily noisy. We use `MaybeUninit` elsewhere for out parameters too

Reply:
Also it somewhat hides the fact that this is an out parameter. Could as well be an in-out parameter from your code, but what we really expect here is to provide some memory to the function that is then filled/initialized.

Reply:
Fine by me!